final/src/main/java/com/example/guestbook/Greeting.java

```diff
@@ -35,8 +36,7 @@
  * want to index using @Index.  This is often a huge win in performance -- though if you don't Index
  * your data from the start, you'll have to go back and index it later.
  *
- * NOTE - all the properties are PUBLIC so that can keep this simple, otherwise,
- * Jackson, wants us to write a BeanSerializaer for cloud endpoints.
+ * NOTE - all the properties are PUBLIC so that can keep the code simple.
```

Comment:
grammar (all the properties are public to keep the code simpler). 

Also, please clean up this comment.
1. Mention of OfyHelper should be a javadoc link
1. Clean up the sentence structure, ex: Must be need to register it with OfyHelper. (I don't like the -- very important)
1. "huge win in performance" - likely more considerations on when to index or not, this comment could lead to someone choosing to index everything.